app/app.component.ts

```diff
@@ -12,7 +12,11 @@ import {About} from './sections/about/about.component';
   selector: 'gapminder-app',
   encapsulation: ViewEncapsulation.None,
   directives: [Header, Footer, ...ROUTER_DIRECTIVES],
-  styles: [ <string> require('./main.styl')],
+  styles: [
+    <string> require('./main.styl'),
+    <string> require('./fonts.styl'),
+    <string> require('./variables.styl')
```

Comment:
@dydek but from the other side we follow Folders-by-Feature convention. And from this point of view root folder is a "whole application feature"  that has it's own styles and html markup. Wouldn't you agree that putting this style file under `assets` directory is a bit strange - cause all the other markup files and styles are in their own feature dirs?